pallene/ast.lua

```diff
@@ -72,6 +72,7 @@ declare_type("Exp", {
     Binop      = {"loc", "lhs", "op", "rhs"},
     Cast       = {"loc", "exp", "target"},
     Paren      = {"loc", "exp"},
+    ExtraRet   = {"loc", "call_exp", "i", "n", "_type"}, -- See checker.lua
```

Comment:
Does `_type` need to be a parameter to ExtraRet's constructor? In all other nodes, the `_type` attribute is separately added by checker.lua and is not a parameter to the constructor.

Reply:
It should be possible to remove it from the constructor and add `_type` to the new node immediately after the call to the constructor. It sure makes more sense now that you asked. I'll change this in the next commit.